src/System.IO.Compression/src/System/IO/Compression/DeflateZLib/DeflateStream.cs

```diff
@@ -759,13 +759,13 @@ private async ValueTask DisposeAsyncCore()
                     }
                     finally
                     {
-                        _deflater = null;
-                        _inflater = null;
+                        _deflater = null!;
```

Comment:
All these `!` shouldn't be needed anymore

Reply:
yes